tests/build.snapshot.js

```diff
@@ -7,7 +7,8 @@ const smallBuildCommons = {
 	'polyfills.js': { size: 4620 },
 	'polyfills.js.map': { size: 31760 },
 	'favicon.ico': { size: 15086 },
-	'sw.js': { size: 3330 },
+	'sw.js': { size: 1628 },
+	'workbox-sw.prod.v1.1.0.js': {size: 47026},
```

Comment:
Oops, sry missed that.
Taking feedback from here. A lot of this might be because  workbox automatically brings the functionality of toolbox as well.